lib/auth/init.go

```diff
@@ -637,6 +639,22 @@ func initializeAuthorities(ctx context.Context, asrv *Server, cfg *InitConfig) e
 		return trace.Wrap(err)
 	}
 
+	// Collect CAs from integrations to avoid deleting them.
```

Comment:
I attempted to create a UT for this but `keyStore.DeleteUnusedKeys` has no effect on software keys. I've also tried test with `TELEPORT_TEST_AWS_KMS_ACCOUNT` but `keyStore.DeleteUnusedKeys` does not delete keys created in the last 5 minutes. I might leave as it is without tests